enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/AuthProceduresTest.java

```diff
@@ -391,21 +391,86 @@ public void roleManagement2() throws Exception
     @Test
     public void roleManagement3() throws Exception
     {
-        testCallEmpty( db, adminSubject, "CALL dbms.createUser('Henrik', 'bar', false)", null );
+        testCallEmpty( db, adminSubject, "CALL dbms.createUser('Henrik', 'bar', false)");
         testCallEmpty( db, adminSubject,
-                "CALL dbms.addUserToRole('Henrik', '" + PredefinedRolesBuilder.PUBLISHER + "')", null );
+                "CALL dbms.addUserToRole('Henrik', '" + PredefinedRolesBuilder.PUBLISHER + "')");
         AuthSubject subject = manager.login( "Henrik", "bar" );
         assertEquals( AuthenticationResult.SUCCESS, subject.getAuthenticationResult() );
         testCallEmpty( db, adminSubject, "CALL dbms.addUserToRole('Henrik', '" + PredefinedRolesBuilder.READER + "')",
                 null );
         testSuccessfulWriteAction( subject );
         testSuccessfulReadAction( subject, 4L );
         testCallEmpty( db, adminSubject,
-                "CALL dbms.removeUserFromRole('Henrik', '" + PredefinedRolesBuilder.PUBLISHER + "')", null );
+                "CALL dbms.removeUserFromRole('Henrik', '" + PredefinedRolesBuilder.PUBLISHER + "')");
         testFailWriteAction( subject );
         testSuccessfulReadAction( subject, 4L );
     }
 
+    //----------User suspension scenarios-----------
```

Comment:
Unit tests are missing for activating and suspending users. At least the following tests need to be added:
- Activating an already activated user
- Suspending an already suspended user
- Non-admin activating/suspending other user